mercury/tests/test_event.py

```diff
@@ -223,3 +257,119 @@ def test_update_event(self):
+        self.assertEquals(venue.description, self.test_venue_data_update["description"])
+        self.assertEquals(venue.latitude, self.test_venue_data_update["latitude"])
+        self.assertEquals(venue.longitude, self.test_venue_data_update["longitude"])
+
+    def test_export_all_csv(self):
+        event = self.create_venue_and_event(self.event_name)
+        sensor = self.create_sensor()
+        self.create_measurement(event, sensor)
+
+        response = self.client.post(reverse(self.event_export_all_csv_url))
+
+        self.assertEqual(200, response.status_code)
+
+    def test_export_all_json(self):
+        event = self.create_venue_and_event(self.event_name)
+        sensor = self.create_sensor()
+        self.create_measurement(event, sensor)
+
+        response = self.client.post(reverse(self.event_export_all_json_url))
+
+        self.assertEqual(200, response.status_code)
+
+    # def test_export_csv(self):
+    #     event = self.create_venue_and_event(self.event_name)
+    #     sensor = self.create_sensor()
+    #     measurement = self.create_measurement(event, sensor)
+
+    #     response = self.client.post(
+    #         reverse(self.event_export_csv_url, kwargs={"event_uuid": event.uuid})
+    #     )
+
+    #     self.assertEqual(200, response.status_code)
+
+    def test_export_json(self):
+        event = self.create_venue_and_event(self.event_name)
+        sensor = self.create_sensor()
+        self.create_measurement(event, sensor)
+
+        response = self.client.post(
+            reverse(self.event_export_json_url, kwargs={"event_uuid": event.uuid})
+        )
+
+        self.assertEqual(200, response.status_code)
+
+    def test_export_no_measurements(self):
+        event = self.create_venue_and_event(self.event_name)
+
+        response = self.client.post(
+            reverse(self.event_export_csv_url, kwargs={"event_uuid": event.uuid})
+        )
+
+        self.assertEqual(302, response.status_code)
+
+    def test_export_all_no_measurements(self):
+        self.create_venue_and_event(self.event_name)
+
+        response = self.client.post(reverse(self.event_export_all_csv_url))
+
+        self.assertEqual(200, response.status_code)
+
+    def test_activate_event(self):
+        event = self.create_venue_and_event(self.event_name)
+
+        response = self.client.post(
+            reverse(self.event_activate_url, kwargs={"event_uuid": event.uuid})
+        )
+
+        self.assertEqual(302, response.status_code)
+
+    def test_deactivate_event(self):
+        event = self.create_venue_and_event(self.event_name)
+
+        response = self.client.post(
+            reverse(self.event_activate_url, kwargs={"event_uuid": event.uuid})
+        )
+
+        self.assertEqual(302, response.status_code)
+
+        response = self.client.post(
+            reverse(self.event_deactivate_url, kwargs={"event_uuid": event.uuid})
+        )
+
+        self.assertEqual(302, response.status_code)
```

Comment:
similarly you could assert that the event is no longer the active event

Reply:
updated, thanks!